setup/state/state.go

```diff
@@ -0,0 +1,194 @@
+package state
+
+import (
+	"github.com/go-ini/ini"
+	"github.com/guregu/null"
+	"github.com/pganalyze/collector/setup/log"
+	"github.com/pganalyze/collector/setup/query"
+)
+
+var SupportedLogLinePrefixes = []string{
+	"%m [%p] %q[user=%u,db=%d,app=%a] ",
+	"%m [%p] %q[user=%u,db=%d,app=%a,host=%h] ",
+	"%t:%r:%u@%d:[%p]:",
+	"%t [%p-%l] %q%u@%d ",
+	"%t [%p]: [%l-1] user=%u,db=%d - PG-%e ",
+	"%t [%p]: [%l-1] user=%u,db=%d,app=%a,client=%h ",
+	"%t [%p]: [%l-1] [trx_id=%x] user=%u,db=%d ",
+	"%m %r %u %a [%c] [%p] ",
+	"%m [%p][%v] : [%l-1] %q[app=%a] ",
+	"%m [%p] ",
```

Comment:
Hmm. I wonder if there is a way we could re-use the list from the main collector code here? (not sure if e.g. we could do a `shared` directory that can be used by all binaries, including this one?)

Reply:
@lfittl Oh, good point. I just didn't think of that. No need to rearrange the code--we can import the logs package here I think. The only thing is, the list in `parse.go` puts the prefix that we document as recommended as third in the SupportedPrefixes list. The recommended prefix should be first in the list we present to the user. At first glance, it looks like I should just be able to rearrange SupportedPrefixes in `parse.go`--is there any reason not to do that?

Reply:
No reason - seems good to re-arrange there (or have a separate constant with the "recommended prefix" as an index?)

Reply:
Oh good idea. Having a separate index will be more robust against changes.